packages/storybook/.storybook/main.js

```diff
@@ -12,36 +12,46 @@ module.exports = {
     'storybook-dark-mode/register',
   ],
   webpackFinal: async config => {
+    const coreSrc = path.resolve(__dirname, '../../core/src');
+
     config.resolve.alias = {
       ...config.resolve.alias,
+      // Resolves imports of @backstage/core inside the storybook config, pointing to src
+      '@backstage/core': coreSrc,
+      // Point to dist version of theme and any other packages that might be needed in the future
       '@backstage/theme': path.resolve(__dirname, '../../theme'),
     };
-    config.resolve.modules.push(path.resolve(__dirname, '../../core/src'));
-    config.module.rules.push(
-      {
-        test: /\.(ts|tsx)$/,
-        use: [
-          {
-            loader: require.resolve('ts-loader'),
-            options: {
-              transpileOnly: true,
-            },
+    config.resolve.modules.push(coreSrc);
+
+    // Remove the default babel-loader for js files, we're using ts-loader instead
```

Comment:
so does this mean we also don't have polyfills for any of our stories?

Reply:
We'll have the same as we had for TS I guess, so shouldn't change much

Reply:
yea es6 only

Reply:
@wesleyyee [#655](https://github.com/spotify/backstage/pull/655/files#diff-a8d0e8b0feb26fb10e28b5d4739e7060) :grin: